### PR TITLE
Fix value of "examples" to be array in advertisement of adcloud extension.

### PR DESCRIPTION
**Pull request by jwen-adobe**
This PR links to the issue #355 

@kstreeter @trieloff @cdegroot-adobe @gabebarcelos-adobe.

This PR fixes the "examples" value to be array.

### CODE REVIEW

**Comment by jwen-adobe** [comment]
@cdegroot-adobe Changed, please approve to get this in.

File: extensions/adobe/experience/adcloud/advertisement.schema.json [hunk 1]
```diff
@@ -23,19 +23,41 @@
...
+            "richMediaBanner": "Rich media banner",
+            "newsFeedVideo": "News feed video",
+            "newsFeedDisplay": "News feed display",
+            "HTML5": "HTML5",
+            "inPageVideo": "In page video",
+            "inPageDisplay": "In page display",
+            "facebook":"Facebook",
+            "twitter": "Twitter",
+            "linearTv": "linear TV",
+            "vod": "VOD"
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/adcloud/advertisement.schema.json]: Please change to "Video on Demand"

File: extensions/adobe/experience/adcloud/advertisement.schema.json [hunk 2]
```diff
@@ -23,19 +23,41 @@
...
+            "banner": "Banner",
+            "richMediaBanner": "Rich media banner",
+            "newsFeedVideo": "News feed video",
+            "newsFeedDisplay": "News feed display",
+            "HTML5": "HTML5",
+            "inPageVideo": "In page video",
+            "inPageDisplay": "In page display",
+            "facebook":"Facebook",
+            "twitter": "Twitter",
+            "linearTv": "linear TV",
```
Comment (cdegroot-adobe) [inline extensions/adobe/experience/adcloud/advertisement.schema.json]: Capitalize the "L"inear

File: extensions/adobe/experience/adcloud/advertisement.schema.json (hunk 1 above)
Comment (jwen-adobe) [reply extensions/adobe/experience/adcloud/advertisement.schema.json]: @cdegroot-adobe done.

File: extensions/adobe/experience/adcloud/advertisement.schema.json (hunk 2 above)
Comment (jwen-adobe) [reply extensions/adobe/experience/adcloud/advertisement.schema.json]: @cdegroot-adobe All fixed.

**Pull request merged**